packages/subgraph/docker/hardhat/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "hardhat-project",
```

Comment:
Can we use the hardhard project from packages/contracts with minimum effort ? if not create a follow up task